addons/im_livechat/static/tests/helpers/mock_server/models/im_livechat_channel.js

```diff
@@ -42,15 +44,10 @@ patch(MockServer.prototype, "im_livechat/models/im_livechat_channel", {
                 },
             ],
         ];
-        let visitor_user;
-        if (user_id) {
-            const visitor_user = this.getRecords("res.users", [["id", "=", user_id]])[0];
-            if (visitor_user && visitor_user.active && visitor_user !== operator) {
-                // valid session user (not public)
-                membersToAdd.push([0, 0, { partner_id: visitor_user.partner_id.id }]);
-            }
-        } else {
-            membersToAdd.push([0, 0, { partner_id: this.publicPartnerId }]);
+        const visitor_user = this.getRecords("res.users", [["id", "=", user_id]])[0];
+        if (visitor_user && visitor_user.active && visitor_user !== operator) {
+            // valid session user (not public)
```

Comment:
do not use active to check for public, compare against public directly (simplification that we don't use the public group in mock server)